package.json

```diff
@@ -47,6 +47,7 @@
     "debug": "^4.3.4",
     "drupal-jsonapi-params": "^2.0.0",
     "jest-axe": "^6.0.0",
+    "jsona": "^1.11.0",
```

Comment:
this was already installed as a dependency of one of our packages, but i explicitly needed to add it when i switched the stories query to use `deserialize: false`